biome.json

```diff
@@ -0,0 +1,38 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.0.0/schema.json",
+	"organizeImports": {
+		"enabled": true
+	},
+	"files": {
+		"ignoreUnknown": true,
+		"ignore": [
+			"examples/basics/src/layouts/**",
+			"examples/with-nanostores/src/layouts/**",
+			"benchmark/projects/**",
+			"benchmark/results/**",
+			"**/dist",
+			"**/smoke",
+			"**/node_modules",
+			"**/fixtures",
+			"**/vendor",
+			"**/.vercel"
+		]
+	},
+	"linter": {
+		"enabled": false
+	},
+	"formatter": {
+		"lineWidth": 100
+	},
+	"javascript": {
+		"formatter": {
+			"quoteStyle": "single",
+			"trailingComma": "es5"
+		}
+	},
+	"json": {
+		"parser": {
+			"allowComments": true
+		}
+	}
```

Comment:
Can we keep using spaces for JSON like before?

Reply:
Still not :( it's a feature we are developing right now. Is it a deal breaker? If so, we could not format JSON files

Reply:
I would prefer keeping the formatting style as close to before, so I'd vote skipping formatting for now 😬

Reply:
The only reason is for our example templates, I think. Two spaces is generally the default.

Reply:
Package managers usually formats with two spaces when installing new dependencies that updates the `package.json` too.

Reply:
That's fixed now